angular/src/app/app.component.ts

```diff
@@ -1,57 +1,33 @@
-import { Component } from '@angular/core';
-
-// Calcite Components
-import '@esri/calcite-components/dist/components/calcite-icon';
-import '@esri/calcite-components/dist/components/calcite-combobox';
-import '@esri/calcite-components/dist/components/calcite-combobox-item';
-
+import { Component } from "@angular/core";
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  selector: "app-root",
+  templateUrl: "./app.component.html",
+  styleUrls: ["./app.component.css"],
 })
 export class AppComponent {
-  title: string = "Angular";
+  title = "calcite-components-angular-example";
+  sliderValue = 50;
 
-  fields = [
-    {
-      "displayText": "Natural Resources",
-      "value": "natural-resources"
-    },
-    {
-      "displayText": "Agriculture",
-      "value": "agriculture"
-    },
-    {
-      "displayText": "Forestry",
-      "value": "forestry"
-    },
-    {
-      "displayText": "Mining",
-      "value": "mining"
-    },
-    {
-      "displayText": "Business",
-      "value": "business"
-    },
-    {
-      "displayText": "Education",
-      "value": "education"
-    },
-    {
-      "displayText": "Utilities",
-      "value": "utilities"
-    },
-    {
-      "displayText": "Transportation",
-      "value": "transportation"
-    }
-  ];
+  public isLoading: boolean = true;
+
+  ngOnInit() {
+    this.fetch();
+  }
 
-  logSelectedFields(event: Event) {
-    const value = (event.target as HTMLInputElement).value;
-    console.log(value);
+  async fetch() {
+    await new Promise(r => setTimeout(r, 2000));
+    this.isLoading = false;
   }
 
+  onSliderInput(event: Event) {
```

Comment:
Mainly curious - are the slider events typed? I just see a generic `Event` type here.

Reply:
Yeah events aren't typed at the moment, but it looks like Stencil may have added support for that in the current version via https://github.com/ionic-team/stencil/pull/4909

Technically the event's type is [`CustomEvent`](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent), so I'll change it to that here.